src/library_idbfs.js

```diff
@@ -83,7 +83,7 @@ mergeInto(LibraryManager.library, {
         callback(null, db);
       };
       req.onerror = (e) => {
-        callback(this.error);
+        callback(e.target.error || 'unknown error');
```

Comment:
Is the ` || 'unknown error'` needed?

Reply:
I don't know, but that's how it's done in ["IDBStore.js"](https://github.com/emscripten-core/emscripten/blob/cd3596b5d922e2f84956bdf005bb150277f5682d/src/IDBStore.js#L48) so I kept it that way just in case.

Reply:
I think lets not include it unless we know we have to.

Reply:
OK, removed. In theory, the error field can be `null` if there is no error, but if there is no error, then the "error" event shouldn't be fired anyway, so we should be good.